src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -545,11 +545,21 @@ protected function initializeContainer()
                         return $previousHandler ? $previousHandler($type, $message, $file, $line) : false;
                     }
 
+                    $backtrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
```

Comment:
I don't get this comment :/ . I need to call `debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS)` here to get the trace to be added to the array we persist in `appDevDebugProjectContainerDeprecations.log`.